package.json

```diff
@@ -23,8 +27,10 @@
     "lint:prettier": "prettier '**/*.{js,css,md}' --list-different",
```

Comment:
Looks like the changes in here are because of prettier. You can add `json` into both the `lint:prettier` command and the `prettier` command here, so we ensure proper formatting. :smile: